tests/test_examples.py

```diff
@@ -61,8 +61,6 @@
 
 CUDA_EXAMPLES = [
     'air/main.py --num-steps=1 --cuda',
-    xfail_param('baseball.py --num-samples=200 --warmup-steps=100 --num-chains=2 --cuda',
```

Comment:
Will put this back when we have CUDA with parallel chains in. Currently we don't have `--cuda` option for the baseball example, so removing this from the tests.